src/confu/schema/core.py

```diff
@@ -738,7 +793,7 @@ def validate(schema, config, raise_errors=False, log=None, **kwargs):
         return (success, errors, warnings)
 
 
-def apply_default(config, attribute, path):
+def apply_default(config: dict, attribute: Attribute, path: List[str]) -> None:
```

Comment:
@jason20c hm i think those may need to be `list[str]` since List refers to the confu `List` attribute

Reply:
Yes I think you're right, will make the changes.